packages/api/src/routes/medical/patient.ts

```diff
@@ -55,7 +62,6 @@ import { cxRequestMetadataSchema } from "./schemas/request-metadata";
 const router = Router();
 const MAX_RESOURCE_POST_COUNT = 50;
```

Comment:
This sounds like post-count, not count of resources on a POST request. And the request might be a PUT.

Maybe rename to `MAX_RESOURCE_COUNT_PER_REQUEST`?